keyboards/gboards/gergoplex/config.h

```diff
@@ -52,4 +52,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define IS_COMMAND() (get_mods() == (MOD_BIT(KC_LCTL) | MOD_BIT(KC_RCTL)) || get_mods() == (MOD_BIT(KC_LSFT) | MOD_BIT(KC_RSFT)))
 
 #define DEBOUNCE 5
-#define COMBO_TERM 40
+#define COMBO_TERM 400
```

Comment:
* `#define COMBO_TERM 200`
  * how long for the Combo keys to be detected. Defaults to `TAPPING_TERM` if not defined.
```
tmk_core/common/action_tapping.h|22| 13:#    define TAPPING_TERM 200
```
How does it feel like @ 200?
I'm more inclined to reset it to the default by removing this line altogether if it's good enough.